full-service/src/db/assigned_subaddress.rs

```diff
@@ -116,6 +124,64 @@ impl AssignedSubaddressModel for AssignedSubaddress {
         diesel::insert_into(assigned_subaddresses::table)
             .values(&subaddress_entry)
             .execute(conn)?;
+
+        // Find and repair orphaned txos at this subaddress.
+        let orphaned_txos = Txo::list_orphaned(&account_id.to_string(), None, None, None, conn)?;
```

Comment:
This should only be exercised if the account is fully synced - is that enforced on AssignedAddress::create?

Reply:
it doesn't need to be only on fully synced accounts. when this is called it will check any orphaned txos, and the following sync process will now be checking for the newly assigned subaddress

Reply:
Ah got it - nice